lib/naics_mapper.rb

```diff
@@ -0,0 +1,37 @@
+require 'csv'
+
+class NaicsMapper
+  CSV_HEADERS = [:code_2007, :title_2007, :code_2012, :title_2012]
+
+  def initialize
+    options = {
+      user_provided_headers: CSV_HEADERS,
+      headers_in_file:       false,
+    }
+    @lookup_hash = SmarterCSV.process("#{Rails.root}/data/2012_to_2007_NAICS.csv", options)
+  end
+
+  def lookup_naics_code(code)
+    Rails.cache.fetch("#{code}", expires_in: 60.minutes) do
```

Comment:
You're creating one instance of the class for use over the lifetime of the importer, and it gets it's data from a file on the local filesystem, which is read one time upon instantiation and then stored in memory...so I don't see the need to use the Rails cache here.